modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/ContentDashboardAdminPortlet.java

```diff
@@ -84,7 +84,8 @@
 		"javax.portlet.portlet-mode=text/html",
 		"javax.portlet.preferences=classpath:/META-INF/portlet-preferences/default-portlet-preferences.xml",
 		"javax.portlet.resource-bundle=content.Language",
-		"javax.portlet.security-role-ref=power-user,user"
+		"javax.portlet.security-role-ref=power-user,user",
+		"javax.portlet.version=3.0"
```

Comment:
What is this for? I think we can remove it.

Reply:
That line was added due an error thrown using the ResourceURLBuilder:

`ERROR [http-nio-8080-exec-2][PortletRequestDispatcherImpl:304] Unable to dispatch request: java.lang.RuntimeException: java.lang.UnsupportedOperationException: Requires 3.0 opt-in`

I'll send you more info via chat